plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/StandardColumnMappings.java

```diff
@@ -457,17 +461,53 @@ public static ColumnMapping timestampColumnMapping()
 
     public static ColumnMapping timestampColumnMapping(TimestampType timestampType)
     {
-        checkArgument(timestampType.getPrecision() <= MAX_SHORT_PRECISION, "Precision is out of range: %s", timestampType.getPrecision());
-        return ColumnMapping.longMapping(
+        if (timestampType.getPrecision() <= MAX_SHORT_PRECISION) {
+            return ColumnMapping.longMapping(
+                    timestampType,
+                    timestampReadFunction(timestampType),
+                    timestampWriteFunction(timestampType));
+        }
+        return ColumnMapping.objectMapping(
```

Comment:
This is in `Add timestamp verification functions` commit, but the commit scope has been extended.

Rename to `Add timestamp mapping functions for high precisions` or squash into the second commit

Reply:
Done!